packages/admin-ui/src/radio/radio.style.ts

```diff
@@ -0,0 +1,84 @@
+import { focusVisible, style } from '@vtex/admin-ui-core'
+import { get } from '@vtex/admin-ui-util'
+
+export const checkmark = style({
+  content: '""',
+  position: 'absolute',
+  display: 'block',
+  borderRadius: 'circle',
+  top: '0.3rem',
+  size: '0.5rem',
+  bg: '$form.control',
+})
+
+export const disabled = style({
+  ':disabled': {
+    bg: '$disabled',
+    border: '$disabled',
+    color: '$disabled',
+    cursor: 'not-allowed',
+    '+ div': {
+      label: {
+        cursor: 'not-allowed',
+      },
+    },
+  },
+})
+
+export const baseline = style({
+  ...focusVisible('neutral'),
+  size: '1.25rem',
+  padding: '6px',
+  bg: '$form.control',
+  border: '$form.control',
+  color: '$form.control',
+  appearance: 'none',
+  position: 'relative',
+  borderRadius: 'circle',
+  cursor: 'pointer',
+  display: 'flex',
+  margin: 0,
+  alignItems: 'center',
+  justifyContent: 'center',
+
+  ':hover:not(:disabled)': {
+    bg: '$form.controlHover',
+    border: '$form.controlHover',
+  },
+  ...disabled,
+})
+
+export const checked = style({
+  ...focusVisible('main'),
+  bg: '$form.controlChecked',
+  color: '$form.controlChecked',
+  border: '$form.controlChecked',
+  position: 'relative',
+
+  '&:after': {
+    ...checkmark,
+    bg: (theme) => get(theme, 'fg.form.controlChecked', ''),
```

Comment:
Just for context, why is this color gotten from style's callback `theme`?

Reply:
We use `fg` tokens in the `color` property since this prop applies color to foreground elements (text, icons, etc.) and we use `bg` tokens in the `background-color` property because this prop applies color to the background. Saying that, you can think of the checkmark as a foreground element in theory, but in practice, we create it by using the `:after` selector, so we have to apply an `fg` token to the `background-color`. 

Let me know if something isn't clear in the explanation and I can try explain it better

Reply:
Got it! Makes sense now